libqtile/widget/spacer.py

```diff
@@ -30,16 +30,28 @@
 
 class Spacer(base._Widget):
     """
-        Just an empty space on the bar. Often used with width equal to
-        bar.STRETCH to push bar widgets to the right edge of the screen.
+        Just an empty space on the bar. Often used with length equal to
+        bar.STRETCH to push bar widgets to the right or bottom edge of the
+        screen.
+
+        Can be installed both in horizontal and vertical bars.
     """
-    def __init__(self, width=bar.STRETCH):
+    def __init__(self, length=bar.STRETCH, width=None):
         """
-            - width: Width of the widget.
-              Can be either ``bar.STRETCH`` or a width in pixels.
+            - length: Length of the widget.
+              Can be either ``bar.STRETCH`` or a length in pixels.
+            - width: DEPRECATED, same as ``length``.
         """
-        base._Widget.__init__(self, width)
+        # 'width' was replaced by 'length' since the widget can be installed in
```

Comment:
Sure! I assume you mean in _addition_ to the comments etc., not in _alternative_, right?